isubit/luggage_events
# LUGG-659 Added Alternate URL field and ds settings

Added _New and Improved_ alternate URL field.
## Testing:
1. Pull down my changes.
2. Run drush fra -y and drush cc all
3. Add an event. Does the Alternate URL field appear above the body text field? Add a URL to the field.
4. Look at the full text view you just created. Does the Alternate URL field appear?
5. Go to the events view and verify that you can see the Alternate URL field in the listing of upcoming/archived events.
## Screenshots:

[Full content view](http://i.imgur.com/lUpNjaR.png)
[Search results/teasers](http://imgur.com/SRQxKqH)
## Questions I have:
- Is the alternate URL label text okay? I wanted it to stand out a bit from the Read More module.
- Do you think that the field's function is clear to users? Does it make sense that the field is an unlabelled link?
- Does its location within the luggage_events creation form make sense? Putting it right below the title didn't seem like the right thing to do to me, honestly. It seems like the user wouldn't think about an alternate URL until it came time to add the body text.
